scss/_patterns_navigation.scss

```diff
@@ -34,11 +35,98 @@
   }
 }
 
-@mixin navigation-pattern (
-  $background-color: $color-brand,
+@mixin navigation-sidebar {
+  .p-navigation--sidebar {
+    @include navigation-pattern(
+      $background-color: $color-x-light,
+      $border-color: $color-mid-light,
+      $text-color: $color-dark
+    );
+
+    a {
```

Comment:
I feel it would be better to target a class within patterns in case we have a link that we don't want to be styled as a side navigation item.